plugin/vcs/vcs.go

```diff
@@ -92,11 +103,25 @@ type RepositoryMember struct {
 	RoleProvider Type               `json:"roleProvider"`
 }
 
+// Repository is the API message for  repository info.
+type Repository struct {
```

Comment:
Same here, since it's not unmarshaled from JSON response, we may not need json response

Reply:
The UserInfo and RepositoryMember have similar issues. It will become obvious when we add the GitHub support.

We can do the refactor in a separate PR.

Reply:
I guess we may need a more informative naming to help identifying which struct is marshaled from a HTTP response, and which is composed.